### ISSUE TITLE
Manufacturing state transition

### ISSUE THREAD

**Issue by RJMJJJ** [chipsalliance/caliptra-rtl#408]
In the CAliptra Main Specs Table-11 for state 101b DebugLocked and manufacturing it mentions:
Manufacturing -> insecure state transition And
Manufacturing -> secured state

# Can you confirm if Manufacturing -> insecure state refers to 
DebugLockedManufacturing -> DebugUnlockedManufacturing i.e. 101b --> 001b
# and if Manufacturing -> secured state refers to 
DebugUnlockedManufacturing -> DebugLockedManufacturing i.e. 001b --> 101b

**Comment by calebofearth** [comment]
This is correct. Insecure refers to states with debug access, secure means debug is locked.

**Comment by RJMJJJ** [comment]
Thanks
